web/js/reminder.js

```diff
@@ -53,8 +53,7 @@ var timerFunction = function (inputDate , id ) {
         displaytime.text("time left : " +diffhours+ "hrs");
     }
     else if(diffhours < 1 && diffminutes > 5){
-        displaytime.css("display" , "block");
-        displaytime.text("time left : " +diffminutes+ "mins");
+
```

Comment:
I know that this for learning the git pull request process. but please put back the code in another pull request :)